Dockerfiles/agent/README.md

```diff
@@ -8,6 +8,7 @@ The following environment variables are supported:
 
 - `DD_API_KEY`: your API key (**required**)
 - `DD_HOSTNAME`: hostname to use for metrics
+- `DD_TAGS`: host tags, separated by spaces. For example: `simple-tag-0 tag-key-1:tag-value-1`
```

Comment:
what do you think about documenting this change in [docs/agent/config.md](https://github.com/DataDog/datadog-agent/blob/6.0.0-beta.8/docs/agent/config.md) also?

Reply:
The space-separation is only for envvars, `datadog.yaml` keeps the yaml-array format. This is due to how viper parses stringarrays in envvars